lib/drives/models/drive.dart

```diff
@@ -57,4 +59,25 @@ class Drive extends Model {
   String toString() {
     return 'Drive{id: $id, from: $start at $startTime, to: $end at $endTime, by: $driverId}';
   }
+
+  static Future<Drive?> userAlreadyHasDrive(
```

Comment:
not sure about the naming? the name sounds like it returns a boolean

Reply:
i wasn't sure too and changed the name, better now?